src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -301,6 +301,36 @@ public void CheckSuccess()
             Assert.Equal("x", result.TopLevelIdentifiers.First());
         }
 
+        [Fact]
+        public void CheckSuccess2()
+        {
+            var config = new PowerFxConfig();
+            var engine = new RecalcEngine(config);
+
+            var result = engine.Check("T.Var = 23", new RecordType()
+                .Add(new NamedFormulaType("T", new RecordType().Add(new NamedFormulaType("Var", FormulaType.String)))));
+
+            Assert.True(result.IsSuccess);
+            Assert.Equal(1, result.Errors.Count(x => x.IsWarning));
+        }
+
+        [Theory]
+        [InlineData(@"""abc"" = 23")]
+        [InlineData(@"23 = ""abc""")]
+        public void CheckSuccess3(string test)

```

Comment:
It may be good to also add as txt - 
although the C# is still good because it lets us verify there is indeed a warning present. (txt only whos errors)